source/vistas/ui/controls/gl_canvas.py

```diff
@@ -67,6 +68,15 @@ def OnPaint(self, event):
     def OnEraseBackground(self, event: wx.EraseEvent):
         pass  # Ignore this event to prevent flickering on Windows
 
+    def UpdateRaycaster(self, event: wx.MouseEvent):
+        if self.auto_raycast:
+            size = self.GetSize()
+            mouse_x = event.GetX() / size.x * 2 - 1
+            mouse_y = - event.GetY() / size.y * 2 + 1
+            self.camera.raycaster.set_from_camera((mouse_x, mouse_y), self.camera)
+            intersects = self.camera.raycaster.intersect_objects(self.camera)
+            print(intersects)
```

Comment:
Will do. Really, this doesn't do anything useful except be used for debugging. It could be used for click-drag controls, but otherwise this code could just be removed.